grails-app/conf/BuildConfig.groovy

```diff
@@ -50,7 +50,7 @@ grails.project.dependency.resolution = {
     dependencies {
         // specify dependencies here under either 'build', 'compile', 'runtime', 'test' or 'provided' scopes e.g.
         // runtime 'mysql:mysql-connector-java:5.1.29'
-        // runtime 'org.postgresql:postgresql:9.3-1101-jdbc41'
+        runtime 'org.postgresql:postgresql:9.3-1101-jdbc41'
```

Comment:
@LyndsayLawrence can you explain if this is needed or if I'm doing something weird to get this thing to run?